### PR TITLE
reborn

### PR DESCRIPTION
**Pull request by sudo-do**

### CODE REVIEW

**Comment by sudo-do** [comment]
i have to check all of those commits be patient @ahihiyou20

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Cheat for owo
 
-* ### [Download Latest Release](https://github.com/sudo-do/discord-selfbot-owo-bot/releases/latest)
+* ### [Download Latest Release](https://github.com/ahihiyou20/discord-selfbot-owo-bot/tags)
```
Comment (sudo-do) [inline README.md:3]: can stay as it is until new STABLE release announced

**Comment by sudo-do** [comment]
seneye görüşürüz

**Comment by ahihiyou20** [comment]
> seneye görüşürüz
What you mean?

**Pull request merged**